.github/workflows/test-origami-component.yml

```diff
@@ -5,22 +5,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    # Required for npx origami-ci branch, specifically for bundle-size-cli
-    - run: git fetch --prune --unshallow
     - uses: actions/setup-node@v2.1.5
       with:
         node-version: 12.x
-    - run: npm install --only=dev
-    - name: Repository name
-      id: repo_name
-      run: echo ::set-output name=REPO_NAME::${GITHUB_REPOSITORY/*\//}
-    - uses: jwalton/gh-find-current-pr@v1
-      id: findPr
-      with:
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-    - run: npx origami-ci branch
-      env:
-        ORIGAMI_CI_NAME: ${{ steps.repo_name.outputs.REPO_NAME }}
-        ORIGAMI_CI_VERSION: 0.0.0
-        ORIGAMI_PULL_REQUEST: ${{ steps.findPr.outputs.pr }}
-        GITHUB_TOKEN: ${{ secrets.ORIGAMI_FOX_ACCESS_TOKEN }}
+    - run: npm install -g npm@^7
+    - run: npm install -g origami-build-tools@prerelease
```

Comment:
Note: We should see if there is any work left before releasing obt v11

Reply:
Should we update this to a specific beta until then?

Reply:
We released v11 and updated the workflow to use v11 👍